### PR TITLE
DDF-1854 Fix caching issue when tags are not specified

### PR DESCRIPTION
**Pull request by clockard**
This is the three line fix for the search cache issue found with DDF-1854
@pklinef 
@tbatie

### CODE REVIEW

**Comment by tbatie** [comment]
:+1:

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java [hunk 1]
```diff
@@ -1044,9 +1044,9 @@ public T propertyIsFuzzy(String propertyName, String literal) {
      * @param literal      value to compare
      * @return result of fuzzy operation
      */
-    public T propertyIs(String propertyName, Object literal) {
-        throw new UnsupportedOperationException(
-                "propertyIs(String,Object) not supported by org.opengis.filter.Filter Delegate.");
+    public T propertyIs(String propertyName, Object literal, String origMethod) {
```
Comment (pklinef) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java]: I think we should change `String origMethod` to something like `PropertyOperation type` where PropertyOperation is an enum or the strings should come from public constants so it is easy for a developer overriding this method to switch on the values.

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java [hunk 2]
```diff
@@ -1044,9 +1044,9 @@ public T propertyIsFuzzy(String propertyName, String literal) {
...
+        throw new UnsupportedOperationException(origMethod + " not supported by " + this.getClass()
```
Comment (pklinef) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java]: Would `String.format` be easier to read here?

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java (hunk 1 above)
Comment (clockard) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java]: Added PropertyOperation enum

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java (hunk 2 above)
Comment (clockard) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java]: Switched to String.format

**Pull request merged**